appendix/basic_commands.tex

```diff
@@ -1033,6 +1033,9 @@ \subsection{DEF}
 Only a single variable can be defined as an argument to a function.  The
 defined function can be used anywhere a BASIC function can normally be used.\\
 
+NOTE: The π character represents the number PI and is entered by pressing XXXXXXXXXXXX on the keyboard. If you are not using an official Commander X16 keyboard,
+then you would need to do XXXXXXXXXXXXXXXXX instead.  ***** NOT SURE IF THIS WORTHY TO BE IN A VISUAL /note BLUE BOX OR NOT?\\
+
```

Comment:
Currently I have tips, notes, and footnotes.  I think we should stick to one of those, if we feel this should be added.  Personally, this feels like a footnote, but then we'd have to figure out what text to add the superscript to.  We wouldn't want to add that in the code block.  We may just want to explain this in an earlier section on constants, and leave it out of the `DEF`.

Reply:
Is there a section for "things that are different about the Commander X16 keyboard", that covers things like the Pi symbol, left-arrow, and PETSCII keys? That's probably the place to mention the Pi constant.

Reply:
I just noticed it as a quite particular visual "character" firstly to be considering how to type it in (where is it?)... and then thought about it being fairly unique (on a keyboard).  A footnote to a "special X16 things you won't find elsewhere" type of page as Tom suggests, might be handy?